docs-api/frontity-packages/collections-packages/components.md

```diff
@@ -144,6 +145,38 @@ const Link = ({
 export default connect(Link);
 ```
 
+#### The `link` processor
+
+Frontity provides a `link` processor. The `link` processor works with the `<html2react>` component and can automatically detect `<a>` tags in the page/post content and intelligently convert them into `<Link>` components.
+
+If the `href` attribute of the `<a>` tag is either:
+
+- a relative link, or
+- an absolute link on the same domain as the WordPress data source
+
+then the processor will convert the the `<a>` tag into a `<Link>` component.
+
+The `<Link>` component created by the processor will be modelled on the `<a>` tag and will have properties consistent with it's attributes - e.g. the `link` property of the `<Link>` component will be the same as the `href` attribute of `<a>` tag being replaced. The processor will also convert absolute links on the same domain to be relative links.
```

Comment:
> [...] consistent with **it's** attributes [...]

I'm not sure but is this a typo? :see_no_evil:

Reply:
You're right - I've fixed it: https://github.com/frontity/api-reference/pull/16/commits/ada6dc58780b7d99c171e70405f8ab9277cce775